tests/Elastic.Markdown.Tests/SiteMap/BreadCrumbTests.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using Elastic.Markdown.IO;
+using FluentAssertions;
+using Xunit.Abstractions;
+
+namespace Elastic.Markdown.Tests.SiteMap;
+
+public class BreadCrumbTests(ITestOutputHelper output) : NavigationTestsBase(output)
+{
+	[Fact]
+	public void ParsesATableOfContents()
+	{
+		var doc = Generator.DocumentationSet.Files.FirstOrDefault(f => f.RelativePath == "testing/nested/index.md") as MarkdownFile;
+
+		doc.Should().NotBeNull();
+
+		doc!.Parent.Should().NotBeNull();
+
+		doc.YieldParents().Should().HaveCount(2);
```

Comment:
nit: I think another critical path worth testing is if the page has `0` parents. Nevertheless, thank you for adding a test!